docs/data/date-pickers/custom-field/behavior-button/MaterialDateRangePicker.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+import { Dayjs } from 'dayjs';
+import Button from '@mui/material/Button';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import {
+  DateRangePicker,
+  DateRangePickerProps,
+  DateRangePickerFieldProps,
+} from '@mui/x-date-pickers-pro/DateRangePicker';
+import { useValidation } from '@mui/x-date-pickers/validation';
+import { validateDateRange } from '@mui/x-date-pickers-pro/validation';
```

Comment:
WDYT, should we re-export `useValidation` from `@mui/x-date-pickers-pro/validation`? 🤔 
Would make at least this import cleaner. 🤷 
```suggestion
import { useValidation, validateDateRange } from '@mui/x-date-pickers-pro/validation';
```

Reply:
Gotcha. 👍 
Makes sense. 💯